docs/user/batch-email-notifications.rst

```diff
@@ -0,0 +1,83 @@
+Batch Email Notifications
```

Comment:
Let's move this content to the `user/web-email-notifications.rst` page, as subsections of "Email notifications"

Reply:
Updated!